esp/src/configurationMode.h

```diff
@@ -0,0 +1,59 @@
+#ifndef CONFIGURATIONMODE_H
+#define CONFIGURATIONMODE_H
+
+#include "constants.h"
+#include "EEPROMTools.h"
+
+int mqtt_broker_EEPROM_offset = 250;
```

Comment:
It should be const int. Better place for declaration will be EEPROMTools or constants. It will be easier to check which parts of EEPROM are occupied.